Source/GUI/Qt/checkerwindow.cpp

```diff
@@ -148,9 +148,15 @@ void CheckerWindow::set_web_view_content(QString& html)
     QObject::connect(main_view, SIGNAL(loadProgress(int)), progress_bar->get_progress_bar(), SLOT(setValue(int)));
     QObject::connect(main_view, SIGNAL(loadFinished(bool)), this, SLOT(create_web_view_finished(bool)));
 
+#if QT_VERSION >= 0x050700
+    QUrl url = QUrl("");

```

Comment:
set to autodectection if I undesrstand well, OK but why not keeping html as we know it is html and this is compatibe with all versions in that case (and/or why limiting autodectect to 5.7+)?